### DISCUSSION TITLE
Possible PR for specific boards

### DISCUSSION THREAD

**Discussion by PontusO** [JAndrassy/WiFiEspAT#87] [Ideas]
Hi Juray,
Would you consider a small PR for a specific range of boards ? All our boards require WIFIESPAT2 to be set and not WIFIESPAT1. It would most likely just be a small set of "if defined(CHALLENGER_2040_WIFI_BLE_RP2040) in src/utility/EspAtDrvTypes.h to ensure the proper settings for only these boards.

**Comment by JAndrassy** [comment]
you could add a -D in boards.txt with build.extra_flags

**Reply by PontusO** [reply]
I'll look into that. Thanks.
